.github/workflows/update-pixi-lockfile.yml

```diff
@@ -0,0 +1,39 @@
+name: Update pixi lockfile
+
+on:
+  schedule:
+  # At 04:00 on Monday
+    - cron: "0 4 * * 1"
+  # ...or manually
+  workflow_dispatch:
+
+jobs:
+  pixi-update:
+    runs-on: ubuntu-latest
```

Comment:
there maybe a chiseled or slim package available which we can also use, but its not important, only if that takes time

Reply:
I think you are confusing this with docker images? These are GitHub actions runner labels and all of the available ones (excluding the possibility of self-hosted runners) are listed here: https://docs.github.com/en/actions/writing-workflows/choosing-where-your-workflow-runs/choosing-the-runner-for-a-job#standard-github-hosted-runners-for-public-repositories

Reply:
yes, catched